### PR TITLE
Small Adjustments made

### PR DESCRIPTION
**Pull request by bryanmontalvan**
resolved requested changes for issue [#1529](https://github.com/operate-first/apps/issues/1529)

### CODE REVIEW

**Comment by larsks** [comment]
@bryanmontalvan Your commit messages (and pull requests) should generally have a subject that describes the nature of your changes so that when someone is reviewing the commit history, they have some idea what a particular commit does without having to look at the diff.

Compare:

> Small adjustments made

With:

> Add a note about correctly updating kustomization.yaml

File: docs/content/grafana/add_grafana_dashboard.md [hunk 1]
```diff
@@ -27,11 +27,18 @@ Pick a suitable `metadata.name`, ensure that its unique in the [`grafana/overlay
 
 Save this file under `grafana/overlays/moc/smaug/dashboards/${DASHBOARD_FOLDER}/grafana-dashboard-example.yaml`.
 
+Make sure to create a kustomization inside your '${DASHBOARD_FOLDER}'
+
+```bash
+  cd ${DASHBOARD_FOLDER}
```
Comment (4n4nd) [inline docs/content/grafana/add_grafana_dashboard.md:33]: just for consistency, can you add `$` signs before each command? or remove it from lines 40/41?

**Comment by HumairAK** [comment]
can you rebase/squash etc?

**Comment by larsks** [comment]
@bryanmontalvan did you mean to close this pr?

**Comment by bryanmontalvan** [comment]
@larsks Yes I did, was having some weird issues with this PR. Hopefully this will be the last for my strange git issues where making a new PR simplifies everything

**Pull request closed without merging**